### PR TITLE
CU-86bwj08f2/Landing---What-our-users-say-Section-5 fixed what our users say grid layout

### PR DESCRIPTION
**Pull request by Dallat65**
**Description of PR**

_A brief description of what changes your PR introduces. The better this description, the quicker we can review, exchange
feedback, and merge!_

---

Please check all that apply.

#### Checklist:

- [ ] Rebased with master
- [ ] PR Description provided
- [ ] Did you communicate these changes with the affected teams?
- [ ] Did unit tests/integration tests/E2E tests get properly added/updated?
- [ ] Is this change production-ready including testing, feature toggles, configurations, and deployment considerations?

#### Frontend Checklist:

- [ ] Is this responsive and mobile friendly?
- [ ] Did you follow the [ARIA specs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)

#### Backend Checklist:

- [ ] Does the endpoint return the correct success/error format?

---
**Screenshots**  
_add relevant screenshots here_

---
**Test Coverage**  
_add relevant screenshots here_

### CODE REVIEW

File: client/src/pages/Landing/WhatOurUsersSay/WhatOurUsersSay.module.scss [hunk 1]
```diff
@@ -8,15 +8,11 @@
...
-.clearfix::after {
-  content: '';
-  clear: both;
-  display: table;
+.clearfix {
+  display: grid;
+  gap: 2rem;
 }
+
 .single_feedback {
```
Comment (KristofaJosh) [inline client/src/pages/Landing/WhatOurUsersSay/WhatOurUsersSay.module.scss:16]: use cameCase

File: client/src/pages/Landing/WhatOurUsersSay/WhatOurUsersSay.module.scss [hunk 2]
```diff
@@ -37,19 +33,13 @@
...
   }
 }
 
 @media screen and (min-width: 992px) {
-  .single_feedback_container {
-    width: 33.3%;
+  .clearfix {
+    grid-template-columns: 1fr 1fr 1fr;
   }
 }
```
Comment (KristofaJosh) [inline client/src/pages/Landing/WhatOurUsersSay/WhatOurUsersSay.module.scss:45]: can be achieved with:     

`grid-template-columns: repeat(auto-fit, minmax(min(100%, 260px), 1fr));
`

**Review by KristofaJosh — CHANGES_REQUESTED** [review]
Overall, improve your CSS names to match the divs purpose

File: client/src/pages/Landing/WhatOurUsersSay/WhatOurUsersSay.module.scss (hunk 1 above)
Comment (Dallat65) [reply client/src/pages/Landing/WhatOurUsersSay/WhatOurUsersSay.module.scss:16]: Done

File: client/src/pages/Landing/WhatOurUsersSay/WhatOurUsersSay.module.scss (hunk 2 above)
Comment (Dallat65) [reply client/src/pages/Landing/WhatOurUsersSay/WhatOurUsersSay.module.scss:45]: This looks like magic!
Can you explain how it works?

Comment (Dallat65) [reply client/src/pages/Landing/WhatOurUsersSay/WhatOurUsersSay.module.scss:45]: 260px results in a 4 column layout on the big screen. I used 300px instead

**Comment by KristofaJosh** [comment]
This PR has changes from https://github.com/Ajo-Savings-Project/ajo-savings/pull/30

meet @pelzman to unblock you

**Pull request closed without merging**